api/config_entry_gateways.go

```diff
@@ -284,6 +284,10 @@ type APIGatewayListener struct {
 	Protocol string
 	// TLS is the TLS settings for the listener.
 	TLS APIGatewayTLSConfiguration
+	// Override is the policy that overrides all other policy and route specific configuration
+	Override APIGatewayPolicy `json:",omitempty"`
+	// Default is the policy that is the default for the listener and route, routes can override this behavior
+	Default APIGatewayPolicy `json:",omitempty"`
```

Comment:
These should probably be pointers because they could be `nil`.

Reply:
done!